spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HomeController, type: :controller do
+  describe "#index" do
+    it "renders 'abtion-rails'" do
+      user = FactoryBot.create(:user)
+      request.headers["X-User-Id"] = user.id
+      request.headers["Authorization"] = "Token #{user.authentication_token}"
+
+      get :index, params: { id: user.id }
+
+      expect(response.body).to eq("abtion-rails")
```

Comment:
Check the http_status too maybe?

Reply:
but how can the body be correct if the status is anything but `200` 🤔 
ahhh... this will tell one the status code if it fails 👍

Reply:
`render status: 418, plain: "abtion-rails"`

Reply:
think this is less of a teapot and more of a 218 ;)